CIPs/0018.md

```diff
@@ -0,0 +1,53 @@
+# CIP [0018]: Change group epoch score calculation for voter rewards
+
+- Date: 2020-08-30
+- Author: @zviadm
```

Comment:
hi @zviadm  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.

Reply:
hey, i finally got around to this and have updated the doc to match the new template.